src/main/java/org/kainos/ea/db/ConnectionDAO.java

```diff
@@ -0,0 +1,32 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.client.DatabaseConnectionFailedException;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class ConnectionDAO {
+
+    private final DatabaseConnector databaseConnector = new DatabaseConnector();
+
+
+    public String testDatabaseConnection() throws DatabaseConnectionFailedException {
+
+        try (Connection c = databaseConnector.getConnection(); Statement st = c.createStatement()) {
+
+            ResultSet rs = st.executeQuery(
+                    "SHOW STATUS WHERE Variable_name='Uptime'");
+
+            while (rs.next()) {
```

Comment:
This can be changed to ` if (rs.next())`

Reply:
Done